invokeai/backend/model_management/model_manager.py

```diff
@@ -341,7 +341,7 @@ def __init__(
         self.logger = logger
         self.cache = ModelCache(
             max_cache_size=max_cache_size,
-            max_vram_cache_size=self.app_config.max_vram_cache_size,
+            max_vram_cache_size=self.app_config.vram_cache_size,
```

Comment:
I think we should also add here `lazy_offloading` argument from config as it added

Reply:
Done - glad you noticed this.